deployment/ansible/roles/geotrellis-site.app/meta/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 dependencies:
   - { role: "azavea.git" }
+  - { role: "azavea.ruby", ruby_development: True }
```

Comment:
I think Node.js may be missing here. Jekyll does not seem to run successfully without it.